### DISCUSSION TITLE
Supported Community README Profile?

### DISCUSSION THREAD

**Discussion by ChristopherA** [community/community#28950] [Discussions]
It is important for when people arrive at our github community that we can offer a good introduction on where to go for what (we have 107 repositories). Right now we've pinned the max number of repos, and over time these repos have become less-and-less code, but just a relatively bare repo with just docs for one of our major topic area. But this isn't quite working as we'd like.

A solution for us is a feature already available for users, which is the README that shows up on their user page. In this, you create a repo with the same number as the user, and it just works.

We'd like the same feature, i.e. a README in a repo with the same name as our community, that displays at the top of our community page.

-- Christopher Allen

**Comment by juanpflores** [comment]
If you have a GitHub organization for your community, you can add a README profile for the org with all the details. It works in the same way as the GitHub Profile README. 

🔗 https://github.blog/changelog/2021-09-14-readmes-for-organization-profiles/
